lib/runners/processor/actionlint.rb

```diff
@@ -0,0 +1,61 @@
+module Runners
+  class Processor::Actionlint < Processor
+    SCHEMA = _ = StrongJSON.new do
+      extend Schema::ConfigTypes
+
+      # @type self: SchemaClass
+      let :config, base(
+        target: target,
+      )
+
+      let :issue, object(
+        snippet: string?,
+      )
+    end
+
+    register_config_schema SCHEMA.config
+
+    def self.config_example
+      <<~'YAML'
+        root_dir: project/
+        target:
+          - .github/workflows/one_worflow.yml
+          - .github/workflows/another_worflow.yml
+      YAML
+    end
+
+    def analyze(changes)
+      return Results::Success.new(guid: guid, analyzer: analyzer)
+      stdout, _stderr = capture3!(
+        analyzer_bin,
+        '-format', '{{json .}}',
+        '-shellcheck=',
+        '-pyflakes=',
+        '--',  # do not process any options after this
+        *Array(config_linter[:target]),
+      )
```

Comment:
```suggestion
        *Array(config_linter[:target]),
        is_success: ->(status) { [0, 1].include?(status.exitstatus) },
      )
```

Both the exit status 0 and 1 need to be treated as `success`. See [this description](https://github.com/rhysd/actionlint/blob/main/docs/usage.md#exit-status).

Reply:
Line added, thank you.